lib/core/providers/state/toolbox_state_provider.dart

```diff
@@ -64,6 +66,10 @@ class ToolBoxStateProvider extends _$ToolBoxStateProvider {
         state = state.copyWith(currentTool: ref.read(shapesToolProvider));
         ref.read(canvasPainterProvider.notifier).repaint();
         break;
+      case ToolType.SPRAY:
+        state = state.copyWith(currentTool: ref.read(sprayToolProvider));
+        ref.read(paintProvider.notifier).updateStrokeWidth(10);
```

Comment:
This should be backed up and restored after tool switch. Same logic was implemented in Kotlin.